hcl2template/testdata/build/post-processor_onlyexcept.pkr.hcl

```
@@ -0,0 +1,17 @@
+
+// starts resources to provision them.
+build {
+    sources = [
+        "source.virtualbox-iso.ubuntu-1204"
```

I would also add another source here.
```suggestion
        "source.virtualbox-iso.ubuntu-1204",
        "source.amazon-ebs.ubuntu-1604",
```